src/llama.cpp

```diff
@@ -57,6 +57,12 @@
     #include <io.h>
 #endif
 
+#if __cplusplus >= 202000L
+    #define LU8(x) (const char*)(u8##x)
+#else
+    #define LU8(x) u8##x
```

Comment:
why prefix the macro with `L` ? , that makes it seem like it is a wide string.

Reply:
as in llama, could be `T` as well. I wanted three characters to minimize the risk of collisions